src/Jackett/Content/index.html

```diff
@@ -266,9 +268,9 @@
                     <tr class="unconfigured-indexer-row">
                         <td><a target="_blank" href="{{site_link}}" title="{{description}}">{{name}}</a></td>
                         <td>{{mains_cats}}</td>
-                        <td class="fit"><span title="{{type}}" class="type-{{type}}">{{type_icon_content}}</span></td>

```

Comment:
class="fit" should remain (this one and the one below). If you need a second class for mobile compatibility feel free to add it.